### ISSUE TITLE
As a Developer, I'd like to be able to retrieve data from the site through a JSON API

### ISSUE THREAD

**Issue by robyurkowski** [grounded/afterburnercms#29]
So I'd like to be able to hit the blog with a GET, for example, and get the content of a blog post. Or I'd like to be able to hit images, and get a list of images to insert.

**Comment by parndt** [comment]
Yes, everything that is available as HTML should be available as JSON. That's the dream!

**Comment by knewter** [comment]
This is a big deal, because for instance we have our latest blog posts in our dashboard and we get them from the json feed Just Because It's There.  We obviously could also get from RSS or whatever, but it was really convenient and we likely just wouldn't have bothered if it had taken more than 2 minutes to code up.

**Comment by robyurkowski** [comment]
It's actually a pretty big deal, too, in that it makes WYSIWYG implementation a matter of just hitting up the right json stream (i.e. getting a list of the first N images).

Of course, this is also something that will have added complexity in that we'll have to check permissions. So you might have access to the 'Pages' module, but not the 'Images' module, in which cases Images in your editor might not work.
